lib/jekyll-docs.rb

```diff
@@ -20,17 +18,13 @@ def init_with_program(prog)
       end
 
       def process(opts)
-        Dir.mktmpdir do |dest_dir|
-          options = opts.merge({
-            "serving"     => true,
-            "watch"       => false,
-            "config"      => File.expand_path("../../site/_config.yml", __FILE__),
-            "source"      => File.expand_path("../../site", __FILE__),
-            "destination" => dest_dir
-          })
-          Jekyll::Commands::Build.process(options)
-          Jekyll::Commands::Serve.process(options)
-        end
+        options = opts.merge({
+          "serving"     => true,
+          "watch"       => false,
+          "destination" => File.expand_path("../../site", __FILE__),
```

Comment:
Why does `Dir.mktmpdir` need to stay in here if the site will be served right from the gem, not copied in a temporary directory and then served. Or is there any limitation or bad thing about serving from a gem install directory?

For the `"source"` option, I checked the Jekyll 3.1.6 code for the `Serve` command and it just doesn't needs or use that option, just expects a `"destination"`.